lib/stdlib/test/epp_SUITE.erl

```diff
@@ -113,6 +113,16 @@ include_local(Config) when is_list(Config) ->
         [ FileLine || {attribute,_,file,FileLine} <- List ],
     ok.
 
+file_macro(Config) when is_list(Config) ->
+    DataDir = proplists:get_value(data_dir, Config),
+    File = filename:join(DataDir, "file_macro.erl"),
+    {ok, List} = epp:parse_file(File, [DataDir], [{source_name, "Other source"}]),
```

Comment:
The third argument for `epp:parse_file/3` is not the option list, but the predefined macros. You will need to use `epp:parse_file/2`.

```suggestion
    {ok, List} = epp:parse_file(File, [{includes, [DataDir]},
                                       {source_name, "Other source"}]),
```